HousingSearchApi/V1/Infrastructure/Factories/ProcessesQueryGenerator.cs

```diff
@@ -25,15 +32,22 @@ public QueryContainer Create<TRequest>(TRequest request, QueryContainerDescripto
                 throw new ArgumentNullException($"{nameof(request).ToString()} shouldn't be null.");
             }
 
+
             _queryBuilder
                 .WithWildstarQuery(processListRequest.SearchText,
```

Comment:
I wonder if we need that - for target ID and patch ID we know we need exact matches only